standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnUtils.java

```diff
@@ -168,16 +169,22 @@ public static boolean isTransactionalTable(Map<String, String> parameters) {
    */
   public static boolean isAcidTable(Table table) {
     return TxnUtils.isTransactionalTable(table) &&
-      TransactionalValidationListener.DEFAULT_TRANSACTIONAL_PROPERTY.equals(table.getParameters()
+      DEFAULT_TRANSACTIONAL_PROPERTY.equals(table.getParameters()
```

Comment:
can't we simply call `isAcidTable(table.getParameters())` ? It's a duplicated code. Same in isTransactionalTable.